src/arduino/blinkende_lysdiode/blinkende_lysdiode.md

```diff
@@ -41,33 +41,34 @@ Under ser du Arduino-brettet. Finn frem din Arduino og studer den!
 
 <figure><img src="brett.png" style="width: 500px"></figure>
 
+
 ## Sjekkliste {.check}
 
 + På den ene siden har vi digitale inn- og utganger merket med **DIGITAL (PWM ~ )**.
 + Fra nå av kaller vi inn/utganger for *port*.
 + Den første digitale porten heter **0**.
 + Den siste digitale porten heter **13**.
 + Det er altså totalt 14 digitale porter.
-+ Portene som har **~** ved siden av seg har funksjonaliteten *PWM* (Pulse Width Modulation).
-+ På den andre siden har vi analoge innganger, merket **ANALOG IN**.
-+ Første analoge inngang heter **A0**.
-+ Den siste analoge inngangen heter **A5**.
-+ Det er altså 6 analoge innganger.
-+ **5V** er 5 Volt.
-+ **GND** er jord, altså 0 Volt eller minus som noen kaller det.
-+ Hva tror du **3.3V** er?
++ **GND** er ground, jord på norsk.
```

Comment:
Tanken med å minimere snakket om PWM var for å ikke legge for mye inn i første oppgave. Er ett tema som heller burde introduseres senere. Eneste vi sier om PWM er vel noe ala: "du må bruke en pin merket med ~ for denne utfordringen"